gsy_framework/constants_limits.py

```diff
@@ -68,6 +68,7 @@ class GeneralSettings:
         SDK_COM_QUEUE_NAME = "sdk-events-responses"
         CN_JOB_QUEUE_NAME = "canary_network"
         SIM_JOB_QUEUE_NAME = "exchange"
+        SIM_JOB_PAID_QUEUE_NAME = "exchange-paid"
```

Comment:
I suggest to add this to the `QueueNames`
after I push https://github.com/gridsingularity/gsy-framework/pull/447/files#diff-c03471d59fe941f2ec19104cce4bfd7dacec0078e4d1919179b9cc4244844f94R247

Reply:
Did this already, however for the "reading from env variables" part I will add it in the separate ticket.